### PR TITLE
Add queries and mutations for manager review

### PR DESCRIPTION
**Pull request by kgolezardi**

### CODE REVIEW

**Comment by kgolezardi** [comment]
Isn't it better to use the term `managerial` instead of `manager`? For example `ManagerialPersonReview`. I'm not sure I'm happy with the new model and API call names... (To be honest, even the term `managerial` doesn't solve the issue for me)

File: review/core/schema/save_manager_person_review_mutation.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+class SaveManagerPersonReviewMutation(WithViewer, ClientIDMutation):
+    class Input:
+        reviewee_id = graphene.NonNull(graphene.ID)
+        sahabiness_rating = Evaluation()
+        problem_solving_rating = Evaluation()
+        execution_rating = Evaluation()
+        thought_leadership_rating = Evaluation()
+        leadership_rating = Evaluation()
+        presence_rating = Evaluation()
+        overall_rating = Evaluation()
```
Comment (smmoosavi) [inline review/core/schema/save_manager_person_review_mutation.py:21]: قرار بود اورال داشته باشیم؟

Comment (kgolezardi) [reply review/core/schema/save_manager_person_review_mutation.py:21]: Yes. We have 4 tabs: Competencies, strengths and weaknesses, projects, and overall. You told me to put the overall rating into `ManagerPersonReview`; which I agree :D

File: review/core/schema/manager_project_comment_query.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    get_or_create_manager_project_comment
+from core.schema.enums import Evaluation
+from graphql_api.schema.extension import Extension
+from .project_review_query import ProjectReviewNode
+from ..models import ProjectComment
+
+
+class ManagerProjectCommentNode(DjangoObjectType):
+    class Meta:
+        model = ProjectComment
```
Comment (smmoosavi) [inline review/core/schema/manager_project_comment_query.py:15]:
```suggestion
        model = ManagerProjectComment
```

**Pull request merged**